hw1/02-palindrome.js

```diff
@@ -1,3 +1,30 @@
-const elem = document.querySelector('input');
+const inputElement = document.querySelector("input");
 
-elem.addEventListener('input', handleInput);
+inputElement.addEventListener("input", isNumPalindrome);
+
+function isNumPalindrome() {
+  var num = document.getElementById("number").value;
+
+  result.textContent = "";
+
+  if (num === "") {
+    return;
+  }
+
+  if (num >= 0) {
```

Comment:
To simplify your code even further, a trick I like to do is reverse the conditional when code is nested. So, in this case, num < 0 is checked. Then I'm going to move the else code and return out of this function, so it doesn't execute the rest of the code. This change will unnest the code and use less brackets.

`if (num < 0) {
  // execute error message
  return;
}

if (parseIntnum) === reversednum)
 // do the rest, and the code is no longer nested`